android/app/build.gradle

```diff
@@ -75,6 +76,9 @@ android {
 }
 
 dependencies {
+
+    // The version of react-native is set by the React Native Gradle Plugin
+    implementation('com.facebook.react:react-android')
```

Comment:
Same as above, let's drop this for now, and reconsider later, but even if we end up using it, it would be in the SDK gradle, not the app.